src/common/queryBuilder.js

```diff
@@ -45,28 +45,54 @@ function buildQuery({ analyzers, fields, operator }) {
          * @returns {object} The resulting query bool object for ElasticSearch.
          */
         queryFn: (query) => {
+            const inclusions = [];
+            const exclusions = [];
+
+            // separate query into inclusions and exclusions
+            // based on hyphen indicator
+            if (query.match(/-/g)) {
+                query.split(" ").forEach((q) => {
+                    if (q.match(/^-/g)) {
+                        exclusions.push(q.slice(1));
+                    } else {
+                        inclusions.push(q);
+                    }
+                });
+            } else {
+                inclusions.push(query);
+            }
+
             // build array of match queries for each analyzer,
             // then flatten them into a single array of queries
-            const queries = analyzers
+            const includedQueries = analyzers
+                .map((analyzer) =>
+                    buildMatchQueries({
+                        analyzer,
+                        fields,
+                        operator: operator || "or",
+                        query: inclusions.join(" "),
+                    }),
+                )
+                .flat();
+            const excludedQueries = analyzers
                 .map((analyzer) =>
                     buildMatchQueries({
                         analyzer,
                         fields,
                         operator: operator || "or",
```

Comment:
One last change, sorry I didn't catch this earlier! This one should always be "or" because we don't ever want to only exclude those containing _all_ excluded terms.

In other words, if there's a document with "samuel beckett" and another "samuel beckett letters", and we searched `samuel -beckett -letters`, even if we used the AND search we would want to exclude both of those because they contain `beckett`, not just the second one (which happens to contain both of our excluded terms).